front/nuxt.config.ts

```diff
@@ -32,24 +31,24 @@ export default defineNuxtConfig({
     components: [
         {
             path: '~/components',
-            pathPrefix: false,
+            pathPrefix: false
```

Comment:
This project really needs some Prettier configs. Formatting is all over the place and it was tricky to minimize changes from just saving files

Reply:
Added basic prettier config to main branch.